pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -103,6 +103,82 @@ func (db *DB) Migration() *migrate.Migration {
 					)`,
 				},
 			},
+			{
+				// some databases may have already this done, although the version may not match
+				Description: "Extending bandwidth_agreements table and drop bwusagetbl",
+				Version:     1,
+				Action: migrate.Func(func(log *zap.Logger, db migrate.DB, tx *sql.Tx) error {
+					v1sql := migrate.SQL{
+						`ALTER TABLE bandwidth_agreements ADD COLUMN uplink BLOB`,
+						`ALTER TABLE bandwidth_agreements ADD COLUMN serial_num TEXT NOT NULL DEFAULT ""`,
+						`ALTER TABLE bandwidth_agreements ADD COLUMN total INT(10)`,
+						`ALTER TABLE bandwidth_agreements ADD COLUMN max_size INT(10)`,
+						`ALTER TABLE bandwidth_agreements ADD COLUMN created_utc_sec INT(10)`,
+						`ALTER TABLE bandwidth_agreements ADD COLUMN expiration_utc_sec INT(10)`,
+						`ALTER TABLE bandwidth_agreements ADD COLUMN action TEXT`,
```

Comment:
Why is this as text instead of a number?

Reply:
received as PUT/GET/GET_AUDIT/GET_REPAIR/PUT_REPAIR by the storage node from uplink/payer

Reply:
PayerBandwidthAllocation.Action is an in32 (BandwidthAction https://github.com/storj/storj/blob/c3d3f41d30f7eca3ecbe449985d67e817257dae4/pkg/pb/piecestore.pb.go#L28)

Reply:
Changed it to INT(10).